onmt/inputters/corpus.py

```diff
@@ -333,15 +348,19 @@ def build_sub_vocab(corpora, transforms, opts, n_sample, stride, offset):
                 continue
             src_line, tgt_line = (maybe_example['src']['src'],
                                   maybe_example['tgt']['tgt'])
+            src_line_pretty = src_line
```

Comment:
Maybe something like "with_feats" would be more explicit? Or "pretty_feats"?

Reply:
As features are optional, there might be cases where it will be just the `src_line`